Jenkinsfile

```diff
@@ -31,9 +31,10 @@ void job_step_update(def value=currentBuild.currentResult) {
     jobStatusUpdate(job_status_internal, env.STAGE_NAME, value)
 }
 
-// For master, this is just some wildly high number
-// For release branches, its the subsequent DAOS version
-next_version = '2.7.0'
+// Should try to figure this out automatically
+/* groovylint-disable-next-line CompileStatic, VariableName */
+String base_branch = 'release/2.6'
+String next_version = base_branch
```

Comment:
Similar to daily-testing, I don't understand how `2.6` is the next version after `2.6`

Reply:
The daosLatestVersion() trusted-pipeline-lib function will return:
- `1000` for `master` or `null`
- `<x>,<y+1>` for `release/<x>.<y>`

So, in this case it returns `2.7`, which is the next version for `release/2.6`

Reply:
Approving, but I'm not a fan of this approach, as I mentioned in https://github.com/daos-stack/daos/pull/14419#pullrequestreview-2074552471